package.json

```diff
@@ -9,7 +9,9 @@
     "lint": "next lint",
     "check": "next lint && tsc --noEmit",
     "prepare": "husky",
-    "test": "vitest"
+    "test": "vitest",
+    "inject": "posthog-cli sourcemap inject --directory ./.next/static/chunks",
```

Comment:
Lets use more intention revealing names. Example: `posthog-inject`, `posthog-upload`. Also use these scripts in CI instead of running the commands separately. This will ensure a single source of truth for these commands and we don't need to fix in multiple places in future if required.